packer/install-dependencies.sh

```diff
@@ -375,7 +375,7 @@ for KUBERNETES_VERSION in ${K8S_VERSIONS}; do
     if [[ $KUBERNETES_VERSION == *"azs"* ]]; then
       HYPERKUBE_URL="msazurestackdocker/hyperkube-amd64:v${KUBERNETES_VERSION}"
     else
-      HYPERKUBE_URL="k8s.gcr.io/hyperkube-amd64:v${KUBERNETES_VERSION}"
+      HYPERKUBE_URL="mcr.microsoft.com/k8s/core/hyperkube-amd64:v${KUBERNETES_VERSION}"
```

Comment:
We should decide if this should be the MCR registry or the new ACR registry. Technically its not supposed to live in MCR.